stl/inc/iomanip

```diff
@@ -208,8 +204,6 @@ struct _Quote_out { // store pointer/length for string
     _Sizet _Size; // length of string
     _Elem _Delim; // delimiter element
     _Elem _Escape; // escape element
-
-    _Quote_out& operator=(const _Quote_out&) = delete;

```

Comment:
Meh, I didn't really think this one through. The other cases where I'm removing deleted assignment operators have no functional changes, since they're implicitly deleted anyway due to reference members. I'll restore this assignment operator and explicitly-default the copy constructor to be consistent with the lack of functional changes in this PR.

(If I were implementing this from scratch, I'd do a lot differently.)